packages/actor-tests/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "resolveJsonModule": true
   },
   "include": [
-    "./lib",
+    "./lib/**/*.ts",
```

Comment:
This is wierd why is this necessary?

Reply:
I was seeing compilation warnings.

Reply:
Odd.   I got no guesses on how this change would change the behavior of typescript